tests/providers/google/cloud/hooks/test_cloud_sql.py

```diff
@@ -1210,32 +1198,30 @@ def test_hook_with_not_too_long_unix_socket_path(self, get_connection):
         self.assertEqual('postgres', connection.conn_type)
         self.assertEqual('testdb', connection.schema)
 
-    @mock.patch("airflow.providers.google.cloud.hooks.cloud_sql.CloudSQLDatabaseHook.get_connection")
-    def test_hook_with_correct_parameters_postgres(self, get_connection):
-        uri = "gcpcloudsql://user:password@127.0.0.1:3200/testdb?database_type=postgres&" \
-              "project_id=example-project&location=europe-west1&instance=testdb&" \
-              "use_proxy=False&use_ssl=False"
+    def _verify_postgres_connection(self, get_connection, uri):
```

Comment:
Personally, I prefer the DAMP principle in testing.
https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
Me too (as you will see in many places below) . But sometimes it's quite ok to add some DRYness to the DAMPness. Especially when you actually test something else those tests are DAMP-first DRY-little.